sickbeard/providers/tntvillage.py

```diff
@@ -425,6 +425,17 @@ def _doSearch(self, search_params, search_mode='eponly', epcount=0, age=0, epObj
                                     logger.log(u"Subtitled, skipping "  + title + "(" + searchURL + ")", logger.DEBUG)
                                     continue
 
+				search_show = re.split(r'([Ss][\d{1,2}]+)', search_string)[0]
```

Comment:
Sorry for the bad indentation, in my source file it was ok, I tought it was only a diff visualization error.